src/styles/pages/_index.scss

```diff
@@ -4,6 +4,5 @@
 @use "cp" as *;
 @use "home" as *;
 @use "visitor-space" as *;
-@use "terms-of-service" as *;
```

Comment:
Any reason for this page to become a module?

Reply:
so that the css lives close to the react component
and so the selectors are hashed, so they cannot leak anymore